src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -1,7 +1,13 @@
+#cython: language_level=3
 from charfbuzz cimport *
 from libc.stdlib cimport free, malloc
 from libc.string cimport const_char
 from typing import Callable, Dict, List, Tuple
+import array
+from cpython cimport array
+
+
+cdef bint PY_NARROW_UNICODE = sizeof(Py_UNICODE) != 4
```

Comment:
Python 3 doesn't need this, do we really need this stuff?

Reply:
it doesn't cost us anything to also support python2 when using cython. We keep writing code using python3 language syntax and semantics and cython does the rest.